automatic/es/update.ps1

```diff
@@ -23,9 +23,9 @@ function global:au_BeforeUpdate {
 
 function global:au_GetLatest {
     $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $re            = "ES-[\d\.]+\.zip"
+    $re            = "ES-[\d\.]+\.x64.zip"
```

Comment:
It looks like there now both x32 and x64 builds of `es`, so the package should be updated to include both, instead of just using the x64.

https://www.voidtools.com/downloads/

Reply:
Indeed, i've tryied, but i don't how this part works : ["(?i)(^\s*FileFullPath\s*=\s*`"[$]toolsDir\\).*"   = "`${1}$($Latest.FileName32)`""](https://github.com/tunisiano187/chocolatey-packages-community/blob/6ae5bc12f8c3a2876e71b132b2a2823a980ae6f2/automatic/es/update.ps1#L9C13-L9C96)
So, i won't be able to do it like you did

Reply:
Maybe this example will help?
https://github.com/TheCakeIsNaOH/chocolatey-packages/blob/master/win-acme/tools/chocolateyinstall.ps1
https://github.com/TheCakeIsNaOH/chocolatey-packages/blob/master/win-acme/update.ps1

Reply:
Thank you